manage

```diff
@@ -54,11 +54,16 @@ restore() {
   echo "Restoring from $dir"
   vols="$dir/vols.tar"
   fakemount="$dir/fakemount.tar"
-  sudo tar -xf $fakemount ./test/fakemount
-  sudo su -c "cd /var/lib/docker/volumes && tar -xf $vols"
+  sudo tar -xspf $fakemount ./test/fakemount
+  sudo su -c "cd /var/lib/docker/volumes && tar -mxspf $vols"
   echo "Done (Restoring from $dir)"
 
   dcup
+
+  sleep 2
+  echo "Hacking SFTPGo 'home' volume permissions"
+  sudo chown $(whoami):$(whoami) /var/lib/docker/volumes/nca_sftpgo-home/_data
```

Comment:
I believe this requires the Docker Compose "project" (which defaults to the containing directory's name) to be named "nca" for the `nca_sftpgo-home` part of this path. Is there a way to adapt this path to reflect the Docker Compose project name? Otherwise I think we'll want to update Docker instructions to warn about this requirement for the project name. Thoughts?

Reply:
For example, on our dev server I have NCA deployed from a directory that matches the sub-URI I have it deployed at `nca-demo`, so the project name matches and the volume is named `nca-demo_sftpgo-home`. I see that the directory is owned by `root:root` but the contents within are owned by `libadmin:libadmin` at the moment.

Reply:
The `manage` script dev/test-only, and is definitely not a good idea to be used for anything else. It isn't documented anywhere except in the "Testing" guide, and even then only as a bit of a footnote: https://uoregon-libraries.github.io/newspaper-curation-app/contributing/testing/#saving-state

If you *want* to use it for a demo site, I'm going to tell you that's probably a terrible idea, but the best approach would be matching the project name (`.env` would be the easiest way to do this, given it's already baked into `env-example`).

Reply:
Okie doke. That's good enough for me. I know there are workarounds, just wanted to check if we should make it more robust or not.